resources/content/db/changelog.xml

```diff
@@ -64,5 +64,7 @@
     <include file="db/core-057.xml"/>
     <include file="db/core-058.xml"/>
     <include file="db/core-059.xml"/>
+    <include file="db/core-060.xml"/>
```

Comment:
Ah, thanks :) was just trying to please @ibuildthecloud who dislikes me using custom names before the final rebase ;)